pkg/framework/api/nodeinfo_podinfo.go

```diff
@@ -263,6 +263,8 @@ type PodInfoMaintainer struct {
 
 	podsWithAffinity             PodInfoSlice
 	podsWithRequiredAntiAffinity PodInfoSlice
+
+	podsByMetaName map[string]*PodInfo
```

Comment:
this is only used by LoadAware plugin ?  I am thinking if we can create separate data structures for plugin specific metrics/info instead of putting all info together in one struct

Reply:
good point, as the maintenance operations of related pods have been delegated to NodeMetricInfo